internal/helm/helm.go

```diff
@@ -152,7 +153,7 @@ func DeployService(logger logr.Logger, parameters ServiceParameters) error {
 		ChartName:   helmChart,
 		Version:     helmVersion,
 		Namespace:   parameters.Namespace,
-		Atomic:      true,
+		Atomic:      parameters.Wait, // implies `Wait true`
```

Comment:
Hm, this means that we are going to have a different behaviour in the two scenarios. If `--wait` then the release will cleaned up for any failures. Otherwise it will be left in a failure state.

Should we just drop the `Atomic` and use only the `Wait`? At least the behaviour with failures will be the same, and more predictable probably.

Reply:
True. Will `service delete` cleanup a failed deployment ?

Reply:
Done with commit 498098b61ff74df303d304caaf3826821c11b1e6